pliers/tests/extractors/test_model_extractors.py

```diff
@@ -447,3 +448,20 @@ def compute_expected_length(stim, ext):
     with pytest.raises(ValueError) as err:
         AudiosetLabelExtractor(top_n=10, labels=labels)
     assert 'Top_n and labels are mutually exclusive' in str(err.value)
+
+def test_spice_extractor():
+    audio_stim = AudioStim(join(AUDIO_DIR, 'homer.wav'))
+    audio_filter = AudioResamplingFilter(target_sr=16000)
+    audio_resampled = audio_filter.transform(audio_stim)
```

Comment:
i've noticed that both here and in the audioset tests `audio_resampled` is not used, my bad. It's no huge deal -- the stimulus does not satisfy model assumptions on the sampling rate, but the extractor does it job, but should probably be fixed.